api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3280,5 +3282,35 @@ public void mergePatients_shouldMoveAllObsWithSameHierarchy() throws Exception {
 		assertEquals(1, encounterService.getEncounter(57).getObsAtTopLevel(false).size());
 		assertEquals(2, encounterService.getEncounter(57).getObsAtTopLevel(true).size());
 	}
+	
+	/**
+	 * Gets the patient, then sees if it can get the patient identifier by the patient program as well
+	 *
+	 * @throws Exception
+	 */
+	@Test
+	public void getPatientIdentifiersByPatientProgram_shouldGetPatientsByIdentifierByProgram() throws Exception {
+
+		executeDataSet(CREATE_PATIENT_WITH_PATIENT_IDENTIFIER_PROGRAM_XML);
+		updateSearchIndex();
+
+		// get the first patient
+		Collection<Patient> panPatients = patientService.getPatients("Pan", null, null, false);
+
+		assertFalse( panPatients.isEmpty(),"The patient List should not be null");
```

Comment:
`assertFalse(panPatients.isEmpty(), "The patient List should not be null");`

spaces here also